### PR TITLE
Specify how this document updates RFC 8999

### LINKED ISSUE #125: Update discipline: tell readers what you update
We have agreed to mark this document as updating RFC 8999. Fine, but this lacks a bit of discipline. We should have a statement inside the text (not just the abstract) explaining in which way RFC 8999 is updated.

### PR DESCRIPTION
**Pull request by huitema**
Add a short text in the introduction section to define how this document updates RFC 8999.

Fixes #125

### CODE REVIEW

File: draft-ietf-quic-version-negotiation.md [hunk 1]
```diff
@@ -65,6 +65,10 @@ receives one. QUIC version 1 {{!QUIC=RFC9000}} allows the server to use a
 Version Negotiation packet to indicate that the version the client chose is
 unacceptable, but doesn't allow the client to safely make use of that
 information to create a new connection with a mutually supported version.
+This document updates {{!QUIC-INVARIANTS=RFC8999}} by defining an alternative
+version negotiation mechanism, while keeping the option to use the
+mechanism defined in {{!QUIC-INVARIANTS=RFC8999}} for negotiating versions
+incompatible with the initially proposed value.
```
Comment (DavidSchinazi) [inline draft-ietf-quic-version-negotiation.md:71]: No need to redefine the references:

```suggestion
This document updates {{QUIC-INVARIANTS}} by defining an alternative version
negotiation mechanism, while keeping the option to use the mechanism defined in
{{QUIC-INVARIANTS}} for negotiating versions incompatible with the initially
proposed value.
```

File: draft-ietf-quic-version-negotiation.md [hunk 2]
```diff
@@ -65,6 +65,10 @@ receives one. QUIC version 1 {{!QUIC=RFC9000}} allows the server to use a
 Version Negotiation packet to indicate that the version the client chose is
 unacceptable, but doesn't allow the client to safely make use of that
 information to create a new connection with a mutually supported version.
+This document updates {{QUIC-INVARIANTS}} by defining an alternative version
+negotiation mechanism, while keeping the option to use the mechanism defined in
+{{QUIC-INVARIANTS}} for negotiating versions incompatible with the initially
```
Comment (DavidSchinazi) [inline draft-ietf-quic-version-negotiation.md:70]: This text contradicts the start of the paragraph, QUIC-INVARIANTS does not define a version negotiation *mechanism*, it only defines a version negotiation *packet*.

File: draft-ietf-quic-version-negotiation.md [hunk 3]
```diff
@@ -65,6 +65,8 @@ receives one. QUIC version 1 {{!QUIC=RFC9000}} allows the server to use a
 Version Negotiation packet to indicate that the version the client chose is
 unacceptable, but doesn't allow the client to safely make use of that
 information to create a new connection with a mutually supported version.
+This document updates {{QUIC-INVARIANTS}} by defining version
+negotiation mechanisms that partially replace the Version Negotiation packet.
```
Comment (huitema) [inline draft-ietf-quic-version-negotiation.md:70]: Simpler text, less likely to contradict other sentences. @DavidSchinazi, what do you think?

File: draft-ietf-quic-version-negotiation.md [hunk 4]
```diff
@@ -65,6 +65,8 @@ receives one. QUIC version 1 {{!QUIC=RFC9000}} allows the server to use a
...
```
Comment (DavidSchinazi) [inline draft-ietf-quic-version-negotiation.md:69]: How about this?
```suggestion
negotiation mechanisms that leverage the Version Negotiation packet.
```

Comment (huitema) [reply draft-ietf-quic-version-negotiation.md]: That works, I merged your commit in the PR.

**Review by DavidSchinazi — APPROVED** [review]

**Comment by DavidSchinazi** [comment]
Thanks for taking the time, Christian! I appreciate it.

**Pull request merged**